Remotion/Data/DomainObjects.UnitTests/ObjectMothers/ExtensibleEnumObjectMother.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using Remotion.ExtensibleEnums;
+
+namespace Remotion.Data.DomainObjects.UnitTests.ObjectMothers
+{
+  /// <summary>
+  /// Provides extensible enum values for unit tests.
+  /// </summary>
+  public class ExtensibleEnumObjectMother
+  {
+    private static readonly Random s_random = new Random();
+
+    /// <summary>
+    /// Gets a random <see cref="ExtensibleEnum{T}"/> value for the given <typeparamref name="T"/>. This is used by unit tests when they need
+    /// code to work with arbitrary extensible enum values. Rather than duplicating the test, once for each value, the test is written once and is executed 
+    /// any value chosen at random.
+    /// </summary>
+    /// <returns>A random <typeparamref name="T"/> value.</returns>
+    public static T GetRandomValue<T> ()
```

Comment:
Our `TestExtensibleEnum` only has two values like `bool`, but I get that it is not the same situation. I will use `Value2()`, since that is not the value provided by any `IPropertyDefaultValueProvider`.